server/app/blocks.go

```diff
@@ -387,3 +409,28 @@ func (a *App) getBoardAndCard(block *model.Block) (board *model.Board, card *mod
 	}
 	return board, card, nil
 }
+
+// extractFileIDsFromBlock extracts file IDs from image and attachment blocks.
+func extractFileIDsFromBlock(block *model.Block) []string {
+	fileIDsToRestore := make([]string, 0, 2)
+
+	if fileIDVal, exists := block.Fields["fileId"]; exists {
+		if fileIDStr, ok := fileIDVal.(string); ok && fileIDStr != "" {
+			fileID := utils.RetrieveFileIDFromBlockFieldStorage(fileIDStr)
+			if fileID != "" {
+				fileIDsToRestore = append(fileIDsToRestore, fileID)
+			}
+		}
+	}
+
+	if attachmentIDVal, exists := block.Fields["attachmentId"]; exists {
```

Comment:
Why can't we remove the attachmentId field right now? I mean, its just about removing this code that checks for attachmentId field.

To clarify, I'm not asking to remove all usages of attachmentId from the whole codebase in this PR, just to not add any new code that references that field.

Reply:
Okay.